img/r/raspberrypi.com.svg

```diff
@@ -0,0 +1 @@
 0 5.378-12.426 20.987 20.987 0 0 0 3.565-8.187c3.279-3.472 4.89-8.954 4.325-14.825Z"/><path d="M82.888 15.646a.507.507 0 0 1 .645.538c-.161 1.517.757 1.326.973 1.057 2.39-2.97 4.92-3.647 7.186-3.447a.507.507 0 0 1 .347.818c-.9 1.335.07 1.589.428 1.35 3.677-2.3 7.185-2.304 8.53-1.338a.507.507 0 0 1 .031.787c-1.36 1.16-.599 1.736-.118 1.56 3.733-1.317 8.886-.151 10.684 1.369a.511.511 0 0 1 .013.773 10.126 10.126 0 0 0-3.898 6.8c-.102.894 1.382.706 2.008.545a.507.507 0 0 1 .632.473c.056 2.15-1.976 4.512-5 6.532-.405.27-.348.92.577.943a.5.5 0 0 1 .43.755c-1.079 2.002-2.545 3.886-7.54 5.18a.512.512 0 0 0-.071.966.49.49 0 0 1 .16.865c-4.974 4.273-17.58 2.589-19.18-4.603a.505.505 0 0 1 .09-.41A47.922 47.922 0 0 1 100.72 21.25a.398.398 0 0 0-.197-.765A34.186 34.186 0 0 0 77.82 34.174a.512.512 0 0 1-.65.147c-8.081-4.172-1.446-16.492 5.718-18.675ZM35.377 35.161a.5.5 0 0 1 .43-.754c.924-.024.982-.674.577-.944-3.024-2.02-5.057-4.383-5-6.532a.507.507 0 0 1 .631-.473c.626.161 2.11.35 2.008-.545a10.126 10.126 0 0 0-3.898-6.8.51.51 0 0 1 .013-.773c1.798-1.52 6.952-2.686 10.685-1.368.48.175 1.242-.402-.119-1.56a.507.507 0 0 1 .03-.788c1.347-.966 4.854-.962 8.531 1.337.36.24 1.329-.014.428-1.35a.507.507 0 0 1 .347-.817c2.266-.2 4.797.476 7.186 3.447.217.27 1.135.46.974-1.057a.507.507 0 0 1 .644-.538c7.164 2.184 13.8 14.503 5.719 18.675a.512.512 0 0 1-.65-.147 34.186 34.186 0 0 0-22.705-13.688.398.398 0 0 0-.197.765 47.921 47.921 0 0 1 20.907 15.907.505.505 0 0 1 .09.411c-1.6 7.192-14.207 8.876-19.18 4.603a.49.49 0 0 1 .16-.865.512.512 0 0 0-.071-.965c-4.995-1.295-6.462-3.18-7.54-5.181Z" class="cls-1"/><path d="M102.704 57.145c2.072 5.513.458 8.163-4.458 5.357a35.232 35.232 0 0 1-10.324-8.654c-3.703-4.4-1.296-6.359 4.537-5.302a15.743 15.743 0 0 1 10.245 8.6ZM80.656 50.943c7.03 5.843-1.024 9.98-9.79 9.98s-16.82-4.137-9.79-9.98a15.63 15.63 0 0 1 19.58 0ZM39.028 57.145a15.743 15.743 0 0 1 10.246-8.599c5.832-1.057 8.239.903 4.536 5.302a35.232 35.232 0 0 1-10.324 8.654c-4.915 2.806-6.53.156-4.458-5.357ZM28.86 87.776a15.738 15.738 0 0 1 1.168-13.392c2.598-4.419 5.212-4.243 6.249-.5a23.48 23.48 0 0 1-1.353 15.476c-1.923 3.823-4.329 2.531-6.064-1.584ZM47.588 116.4a15.645 15.645 0 0 1-11.768-13.963c.975-22.688 33.673 17.829 11.768 13.963ZM48.254 88.143c-5.483-3.166-6.934-10.918-3.24-17.315s11.131-9.017 16.615-5.851c5.483 3.165 6.934 10.918 3.24 17.315s-11.132 9.016-16.615 5.85ZM80.016 125.058a15.658 15.658 0 0 1-18.3 0c-5.433-3.752.033-8.069 9.15-8.069s14.583 4.317 9.15 8.07Z" class="cls-2"/><ellipse cx="70.866" cy="100.396" class="cls-2" rx="12.728" ry="10.909"/><path d="M76.863 82.292c-3.694-6.397-2.243-14.15 3.24-17.315 5.484-3.166 12.923-.547 16.616 5.85s2.242 14.15-3.24 17.316c-5.484 3.165-12.923.546-16.616-5.851ZM94.144 116.4c-21.904 3.866 10.793-36.651 11.768-13.963A15.645 15.645 0 0 1 94.144 116.4ZM112.872 87.776c-1.735 4.115-4.14 5.407-6.064 1.584a23.48 23.48 0 0 1-1.353-15.475c1.038-3.744 3.652-3.92 6.249.5a15.738 15.738 0 0 1 1.168 13.391Z" class="cls-2"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 99 128"><path d="M99 73c0-6-3-11-7-14a17 17 0 0 0-3-6 19 19 0 0 0-7-13 7 7 0 0 0 2-2c4-2 7-6 8-9a13 13 0 0 0 4-9 8 8 0 0 0-1-3 7 7 0 0 0 2-8c-1-3-6-6-13-6a10 10 0 0 0-7-3 16 16 0 0 0-3 1 9 9 0 0 0-4-1 15 15 0 0 0-7 2 7 7 0 0 0-1 0c-3 0-7 2-8 3a24 24 0 0 0-4 5 24 24 0 0 0-5-5c-1-1-4-3-8-3a15 15 0 0 0-8-2 9 9 0 0 0-3 1 16 16 0 0 0-4-1 10 10 0 0 0-7 3C9 3 4 6 2 9a7 7 0 0 0 2 8 8 8 0 0 0 0 3 13 13 0 0 0 3 9c1 3 4 7 9 9a7 7 0 0 0 1 2 19 19 0 0 0-7 13 17 17 0 0 0-3 6c-4 3-6 8-7 14 0 6 1 12 4 15a21 21 0 0 0 4 8 21 21 0 0 0 5 13 22 22 0 0 0 12 7 29 29 0 0 0 10 6 21 21 0 0 0 15 6 21 21 0 0 0 15-6 29 29 0 0 0 9-6 22 22 0 0 0 12-7 21 21 0 0 0 5-13 21 21 0 0 0 4-8c3-3 5-9 4-15z"/><path fill="#46af4b" d="M62 9c0 2 1 1 1 1 3-3 5-3 7-3a1 1 0 0 1 1 1v1c4-2 7-2 9-1-2 1-1 2 0 2 3-1 9 0 10 1a1 1 0 0 1 0 1 10 10 0 0 0-4 7h2a1 1 0 0 1 1 1c0 2-2 4-5 6l1 1a1 1 0 0 1 0 1c-1 2-2 4-7 5a1 1 0 0 0 0 1v1c-5 4-18 3-19-5a48 48 0 0 1 20-16v-1a34 34 0 0 0-22 14 1 1 0 0 1-1 0c-8-4-2-16 6-18zM14 28a1 1 0 0 1 1-1v-1c-3-2-5-4-5-6a1 1 0 0 1 1-1h2a10 10 0 0 0-4-7 1 1 0 0 1 0-1c2-1 7-2 11-1 0 0 1-1-1-2a1 1 0 0 1 1 0c1-1 4-1 8 1V8a1 1 0 0 1 1-1c2 0 5 0 7 3 0 0 1 1 1-1a1 1 0 0 1 1 0c7 2 13 14 5 18a34 34 0 0 0-23-14v1a48 48 0 0 1 21 16c-2 8-14 9-19 5v-1a1 1 0 0 0 0-1c-5-1-7-3-8-5z"/><path fill="#cd2355" d="M81 50c3 6 1 8-4 5a35 35 0 0 1-10-8c-4-5-2-7 4-6a16 16 0 0 1 10 9zm-22-6c7 6-1 10-9 10-9 0-17-4-10-10a16 16 0 0 1 19 0zm-41 6a16 16 0 0 1 10-9c6-1 8 1 5 6a35 35 0 0 1-11 8c-5 3-6 1-4-5zM8 81a16 16 0 0 1 1-14c2-4 5-4 6 0a23 23 0 0 1-1 15c-2 4-5 3-6-1zm18 28a16 16 0 0 1-11-14c1-22 33 18 11 14zm1-28c-5-3-7-11-3-17 3-7 11-9 16-6 6 3 7 11 4 17-4 7-11 9-17 6zm32 37a16 16 0 0 1-19 0c-5-4 1-8 10-8s14 4 9 8z"/><ellipse cx="49.6" cy="93.3" fill="#cd2355" rx="12.7" ry="10.9"/><path fill="#cd2355" d="M56 75c-4-6-3-14 3-17 5-3 13-1 16 6 4 6 3 14-3 17-5 3-13 1-16-6zm17 34c-22 4 11-36 12-14a16 16 0 0 1-12 14Zm19-28c-2 4-4 5-6 1a23 23 0 0 1-2-15c1-4 4-4 6 0a16 16 0 0 1 2 14z"/></svg>
```

Reply:
Updated in a51c061fe48646e14d972eff731572e559fffffb.